### PR TITLE
Add websocketpp

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
@traversaro @wolfv - are you happy to become maintainers of this feedstock?

**Comment by Tobias-Fischer** [comment]
@traversaro - could you please have a look at the Windows build and whether the install location of the cmake script is correct?

**Comment by traversaro** [comment]
> @traversaro @wolfv - are you happy to become maintainers of this feedstock?

Ok!

**Comment by Tobias-Fischer** [comment]
Kind ping @wolfv - would be great if you let me know if you want to become a maintainer (no worries if not).

**Comment by wolfv** [comment]
Yes :)

**Comment by Tobias-Fischer** [comment]
@conda-forge/help-c-cpp this is ready for review. Many thanks!

File: recipes/websocketpp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    sha256: 6ce889d85ecdc2d8fa07408d6787e7352510750daa66b5ad44aacb47bea76755
+    patches:
+      - cmake-configversion-compatibility-anynewerversion.patch
+      - fix-cmake-find-boost.patch
+      - fix-cmake-install-path.patch
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('websocketpp', max_pin='x.x.x') }}
```
Comment (saraedum) [inline recipes/websocketpp/meta.yaml:19]: Why do we need a run exports if this is a header only library?

Comment (Tobias-Fischer) [reply recipes/websocketpp/meta.yaml]: Very true, thanks for the catch!

**Comment by Tobias-Fischer** [comment]
Thanks @saraedum for the review - I fixed the issue that you pointed out.

**Comment by Tobias-Fischer** [comment]
@saraedum - gentle reminder about this PR. Thanks :)

/cc @wolfv

**Pull request merged**